scrunch/datasets.py

```diff
@@ -1441,12 +1480,32 @@ class Variable(object):
 
     CATEGORICAL_TYPES = {'categorical', 'multiple_response', 'categorical_array'}
 
-    def __init__(self, resource, dataset_resource):
-        self.resource = resource
-        self.url = self.resource.self
-        self.dataset = Dataset(dataset_resource)
+    def __init__(self, var_tuple, dataset):
+        self.shoji_tuple = var_tuple
+        self.is_instance = False
+        self._resource = None
+        # pass an Entity class to save time
+        if isinstance(dataset, pycrunch.shoji.Entity):
+            self.dataset = dataset
+        else:
+            # Need to instantiate Dataset for ds.order method
+            self.dataset = Dataset(dataset)
```

Comment:
We need an instance for this particular line: https://github.com/Crunch-io/scrunch/blob/master/scrunch/datasets.py#L1773. It's also not something I added. I was an instance all along.